lib/polka-storage-proofs/src/zero_reader.rs

```diff
@@ -49,6 +54,73 @@ mod tests {
 
     use super::ZeroPaddingReader;
 
+    /// Sole purpose of this reader is to simulate the file reading in the OS.
+    /// When we give it a buf of a certain buf.length(), it might read n < buf.length().
+    struct IOSimulatingReader {
+        read_count: usize,
+        served_data: Vec<Vec<u8>>,
+    }
```

Comment:
```suggestion
    /// Sole purpose of this reader is to simulate the file reading in the OS.
    /// When we give it a buf of a certain buf.length(), it might read n < buf.length().
    struct IncompleteReader {
        read_count: usize,
        served_data: Vec<Vec<u8>>,
    }
```